src/Language/PureScript/Parser/Lexer.hs

```diff
@@ -254,7 +254,7 @@ parseToken = P.choice
 
   parseNumber :: P.Parsec String u (Either Integer Double)
   parseNumber = (consumeLeadingZero >> P.parserZero) <|>
-                  (Right <$> P.try (PT.float tokenParser) <|>
+                  (Right <$> P.try parseDouble <|>
```

Comment:
Would it be worth commenting the ticket number here in case anyone stumbles on it and wonders why Parsec wasn't used?